docs/style-guide.rst

```diff
@@ -931,4 +931,10 @@ built-in or otherwise reserved name.
 General Recommendations
 =======================
 
-TODO
+* Function parameter variable names
+
+It's convention to start function parameter variable names with an underscore (`_`) in order to differentiate them from global variables. e.g. `function myFunction(_foo, _bar) {  }`
+
+* Private functions
+
+It's convention to start private function names with an underscore (`_`). e.g. `_myPrivateFunction`
```

Comment:
I'm not sure about this. Is any project doing this?

Reply:
This is one project: Crypto Zombies
at [lesson 1 chapter 9](https://cryptozombies.io/en/lesson/1/chapter/9)

Reply:
There needs to be a bit stronger concensus if we put into the guide claiming it is a convention.

Reply:
They also took it from the fact that it's a convention in languages like [Python](https://hackernoon.com/understanding-the-underscore-of-python-309d1a029edc), [PHP](https://stackoverflow.com/questions/663350/whats-the-deal-with-a-leading-underscore-in-php-class-methods) and other OOP languages. However in JS since everything is public, there are some [discussions](https://stackoverflow.com/questions/4484424/underscore-prefix-for-property-and-method-names-in-javascript) wether to do that or not. Also seen in the [React and NPM](https://github.com/airbnb/javascript/issues/1024) source code.

But I guess any Styling convention comes down to preference really...

Reply:
And exactly because it comes down to preference I would rather not put naming conventions in a styling guide.

Reply:
I must say the `Function parameter variable names` I have seen in many places and do think it's useful. Maybe we can just scratch out the `private function name` one?

Reply:
The convention in python also comes from the fact that you cannot mark members as private (or at least you were not able to in previous versions). I think PHP is similar.